array-from.js

```diff
@@ -67,6 +67,7 @@ if (!Array.from) {
 				}
 				defineProperty(A, k, {
 					'value': mappedValue,
+					'writeable': true,
```

Comment:
wow I've been spelling it with an e for years.. turns out it's an alternate spelling (http://dictionary.reference.com/browse/writeable?s=t)

and they're both supported

```js
var o = Object.create({}, {ro: {writeable: false}}); o.ro = 1; console.log('writeable', o.ro) //undefined
var o = Object.create({}, {ro: {writable: false}}); o.ro = 1; console.log('writable', o.ro) //undefined
```

Yep I'll add test, and spell it in the more common form